packages/target-ethers-v5/README.md

```diff
@@ -21,3 +21,49 @@
 This package requires TypeScript >= 3.8. If you need support for earlier TS versions check out: 1.0 version of this package.
 
 ## [TypeChain readme](https://github.com/ethereum-ts/TypeChain)
+
+## Contract typings
+
+The main files generated by this target are `<contract-name>.d.ts`. They declare typesafe interfaces for your contracts on top of ethers `Contract` instances:
+* typed contract's methods, available both at `contract.someMethod(...)` and `contract.functions.someMethod(...)`
+* typed events in `contract.interface.events.AnEvent` and filters in `contract.filters.AnEvent`
+* typed method gas estimates in `contract.estimate.someMethod`
+* overrides for the event listener methods (`on`, `once`, etc) that return the same contract type.
+
+Note: these are just _type declarations_ to help you call the blockchain properly, so they're not available at runtime, and all of the contracts are still instances of the same `Contract` class.
+
+## Contract factories
+
+This target also generates a concrete factory class for each contract, to help you deploy or connect to contract instances. The factory classes are an extension of ethers' `ContractFactory`. They serve two main purposes:
+* wrap passing contract ABI and bytecode to the `ContractFactory` class, so you don't have to load and parse the JSON manually
+* provide a correctly typed interface to `ContractFactory` (since it returns plain `Contract` instances).
+
+Abstract contracts or solidity interfaces are handled a bit different, because they have no bytecode. For those, a simplified factory is generated that doesn't extends `ContractFactory`, and only includes the static `connect` method, so you can easily connect to a deployed instance without having to pass the ABI manually.
+
+## Basic example
+
+Suppose you have an `Erc20Token.sol` solidity interface and a `DummyToken.sol` contract implementing it.
+
+```typescript
+import { BigNumber } from 'ethers/utils';
```

Comment:
Can you confirm that this works as of v5? Or otherwise, BigNumber was moved to top bucket, so it can directly be accessed from `ethers`.
```suggestion
import { BigNumber } from 'ethers';
```